github-review.el

```diff
@@ -126,6 +126,16 @@ return a deferred object"
     (let ((query (format "query {
   repository(name: \"%s\", owner: \"%s\") {
     pullRequest(number: %s){
+      reviewThreads(first: 100) { edges { node {comments(first: 10) {
```

Comment:
1.1 reviewThreads view should be sufficient.

Reply:
2.1 reply to the review's 1.1.

Reply:
A 5th(?) 5.0 multi-comment hopefully multi-commit review starts to reply 2.1.